### DISCUSSION TITLE
PaDiM with wide_resnet50_2

### DISCUSSION THREAD

**Discussion by haobo827** [open-edge-platform/anomalib#271] [Q&A] [answered]
Only resnet18 in
model:
  name: padim
  backbone: resnet18
  layers:
    - layer1
    - layer2
    - layer3
  normalization_method: min_max # options: [none, min_max, cdf]
  threshold:
    image_default: 3
    pixel_default: 3
    adaptive: true

If I want to use wide_resnet50_2 as net, how can I write the config.yaml?

**Comment by samet-akcay** [comment]
You need to set `backbone: wide_resnet50_2`

**Comment by haobo827** [comment]
> You need to set `backbone: wide_resnet50_2`

Do other parameters in [model] need to be changed?

**Comment by samet-akcay** [comment]
It depends on what you want, but if you just want to choose `wide_resnet50_2` as the backbone and extract the first three layers, you don't need to change anything.

**Comment by haobo827** [comment, marked as answer]
> It depends on what you want, but if you just want to choose `wide_resnet50_2` as the backbone and extract the first three layers, you don't need to change anything.

That's what I'm thinking at the moment, how nice of you

**Reply by isackf** [reply]
Hi, @samet-akcay, Is there a way to replace/change the extracted feature from layer1, layer2 and layer3 to another specific layer of the CNN backbone? 

> model:
  name: padim
  backbone: resnet18
  layers:
    - layer1
    - layer2
    - layer3

**Reply by djdameln** [reply]
Hi, yes this is possible. Just find the names of the layer(s) from which you want the model to extract the features, and specify them in the list. For example:

```
model:
  name: padim
  backbone: resnet18
  layers:
    - layer3.0
    - layer4
```

**Reply by isackf** [reply]
Got it, Thank you. Amazing work!
